packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -787,11 +807,11 @@ export class InputManager {
 
             // Meshes
             this._pickedDownMesh = null;
-            let pickResult;
-            if (scene.skipPointerDownPicking || (scene._registeredActions === 0 && !scene.onPointerObservable.hasObservers())) {

```

Comment:
> !scene.onPointerObservable.hasObservers()

Is it not necessary to check this anymore?

Reply:
Please confirm. This shouldn't be based on a feeling. ;-)

Reply:
Logic has been modified and we now need that check